posts/2023/mfdataset.ipynb

```diff
@@ -0,0 +1,5675 @@
+{
```

Comment:
<p>Looks like the single file is more like ~2GB now? Or perhaps this is going to be updated with the new CESM2 LE file location</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>yeah not sure what happened. Thanks!</p>